noir-projects/aztec-nr/aztec/src/macros/notes/mod.nr

```diff
@@ -334,6 +330,7 @@ comptime fn generate_multi_scalar_mul(
                 aztec::protocol_types::point::Point { x: $generator_x, y: $generator_y, is_infinite: false }
                 },
             );
+            // TODO: explain why it's ok to use the unsafe `from_field_unsafe`, here.
```

Comment:
Is it ok? Can we not use the fn?

Reply:
I don't know. I couldn't determine whether it was safe or not, so I planted that todo as a flag.